port/unix/omrmmap.c

```diff
@@ -50,11 +49,97 @@
 #include "protect_helpers.h"
 #include "omrportpriv.h"
 
+#if defined(J9ZOS390)
+#include "omrvmem.h"
+#if !defined(MAP_FAILED)
+#define MAP_FAILED ((void*)-1)
+#endif /* !defined(MAP_FAILED) */
+#endif /* defined(J9ZOS390) */
+
 #if defined(AIXPPC)
 #include <sys/shm.h>
 #include <sys/vminfo.h>
 #endif/*AIXPPC*/
 
+#if defined(J9ZOS390)
+static J9MmapHandle *
+omrmmap_read_map_file(struct OMRPortLibrary *portLibrary, intptr_t file, uint64_t offset, uintptr_t size, uint32_t category)
+{
+	/* default implementation will allocate memory and read the file into it */
+	uintptr_t numBytesRead;
+	intptr_t rc;
+	void *mappedMemory;
+	void *allocPointer;
+	J9MmapHandle *returnVal;
```

Comment:
These local variables should have initializers.

Reply:
Fixed.